components/service/glean/docs/pings.md

```diff
@@ -0,0 +1,42 @@
+# glean pings
+
+glean provides a set of pings that are assembled, out of the box, without any developer intervention.
+Every glean ping is in JSON format and contains a [common section](#the-common-information-section)
+with shared information data, included under the `ping_info` key.
+The following is a list of the pings, along with the conditions that triggers them.
+
+- [`baseline` ping](baseline.md)
+- `events` ping (TBD)
+- `metrics` ping (TBD)
+
+## The `common` information section
+The following fields are always included in this section, for every ping.
+
+| Field name | Type | Description |
+|---|---|---|
+| `ping_type` | String | The name of the ping |
+| `app_build` | String | TBD |
+| `telemetry_sdk_build` | String | An identifier for the version of the glean library |
+| `client_id` | String |  A UUID identifying a profile and allowing user-oriented correlation of data |
+| `seq` | Counter | A running counter of the number of times this ping type has been sent |
+| `experiments` | Object | A dictionary of [active experiments](#the-experiments-object) |
+| `start_time` | Datetime | The time of the start of collection of the data in the ping |
+| `end_time` | Datetime | The time of the end of collection of the data in the ping, and ping generation time (which is likely well before ping transmission time) |
+| `profile_age` | Datetime | TBD |
+
+### The `experiments` object
+This object contains experiments, keyed by the experiment `id`. Each listed experiment contains the
+`branch` the client is enrolled in and may contain a string to string map with additional data in the
+`extra` key. Both the `id` and `branch` are truncated to 30 characters.
```

Comment:
Limits like these remind me that there might be implicit datatype limits that might be helpful to link to from this documentation. Maybe a section "These data types are explained in further detail in \[the data types documentation](...). Any limits or exemptions to limits are documented here with the fields that have or are exempt from the limitations."

Reply:
Great point. I just filed [a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1513618) about having in-tree documentation for the different metric types.